packages/tooltip/src/components/tooltip/Tooltip.tsx

```diff
@@ -1,14 +1,89 @@
-import { Text } from "@stenajs-webui/core";
+import { Indent, Row, Text } from "@stenajs-webui/core";
+import {
+  Icon,
+  stenaExclamationCircle,
+  stenaExclamationTriangle,
+  stenaInfoCircle,
+} from "@stenajs-webui/elements";
+import TippyComponent, {
+  TippyProps as TippyComponentProps,
+} from "@tippyjs/react";
+import cx from "classnames";
 import * as React from "react";
-import { Popover, PopoverProps, PopoverVariant } from "../popover/Popover";
+import { useLazyPopover } from "../../hooks/UseLazyPopover";
+import { TippyCallbackRef } from "../../hooks/UseTippyInstance";
+import styles from "./Tooltip.module.css";
 
-export interface TooltipProps extends Partial<Omit<PopoverProps, "content">> {
-  variant?: PopoverVariant;
+type TooltipVariant = "info" | "warning" | "error";
+
+export interface TooltipProps {
```

Comment:
```suggestion
export interface TooltipProps extends TippyProps {
```

And move TippyProps interface above